functions/common.py

```diff
@@ -28,6 +29,60 @@ def factorial(argument: int) -> int:
     return result
 
 
+def mod(dividend: float, divisor: float) -> float:
+    # TODO not sure on the typing for this,
+    # and should check if can be optimized
+    while dividend > divisor:
+        dividend -= divisor
+    while dividend < divisor:
+        dividend += divisor
+    return dividend
```

Comment:
I would just use the built-in mod because it's a built-in operator. Unless Python's mod works differently from this one?